sdk/src/main/java/io/radar/sdk/RadarTrackingOptions.kt

```diff
@@ -116,7 +116,7 @@ data class RadarTrackingOptions(
             useStoppedGeofence = true,
             stoppedGeofenceRadius = 200,
             useMovingGeofence = true,
-            movingGeofenceRadius = 200
+            movingGeofenceRadius = 100
```

Comment:
iOS PR has this at 0 (https://github.com/radarlabs/radar-sdk-ios/blob/6a2a1e17c5abd07ff8074b8918186b2dbf5fd7b8/RadarSDK/RadarTrackingOptions.m#L79) -- should we make it consistent?

Reply:
iOS no longer needs a moving geofence. Low power location manager keeps the process awake while moving, and it just makes things noisy.

On Android in responsive mode, we use the moving geofence with a dwell check to get location updates more frequently than we would otherwise. Fused location delivers updates "a few times per hour," whereas geofencing delivers updates "every couple of minutes" (https://developer.android.com/about/versions/oreo/background-location-limits).